### PR TITLE
gwb-grid: rework help text, add version info

### PR DESCRIPTION
**Pull request by tjhei**
FYI: @mibillen

### CODE REVIEW

File: source/gwb-grid/main.cc [hunk 1]
```diff
@@ -247,14 +248,28 @@ int main(int argc, char **argv)
...
   try
     {
+      if (find_command_line_option(argv, argv+argc, "-v") || find_command_line_option(argv, argv+argc, "--version"))
+        {
+          std::cout << "World Builder Grid Visualization tool.\n"
+                    << "GWB Version: " << WorldBuilder::Version::MAJOR << "."
+                    << WorldBuilder::Version::MINOR << "."
+                    << WorldBuilder::Version::PATCH << "."
+                    << WorldBuilder::Version::LABEL
+                    << std::endl;
```
Comment (MFraters) [inline source/gwb-grid/main.cc:258]: while we are at it, can you also add the git hash?

**Review by gassmoeller — APPROVED** [review]
Looks much cleaner to me.

**Review by MFraters — APPROVED** [review]
Looks great, thanks! One small request for an addition, otherwise good to go.

**Comment by tjhei** [comment]
I will need to update some test output, I think.

File: source/gwb-grid/main.cc (hunk 1 above)
Comment (tjhei) [reply source/gwb-grid/main.cc:259]: done

Comment (MFraters) [reply source/gwb-grid/main.cc:259]: Thanks!

**Pull request merged**